ftest.sh

```diff
@@ -145,21 +145,26 @@ fi
 trap 'set +e; cleanup' EXIT
 
 CLUSH_ARGS=($CLUSH_ARGS)
-
-DAOS_BASE=${SL_PREFIX%/install}
 if ! clush "${CLUSH_ARGS[@]}" -B -l "${REMOTE_ACCT:-jenkins}" -R ssh -S \
     -w "$(IFS=','; echo "${nodes[*]}")" "set -ex
-# allow core files to be generated
+# allow ulimit files
```

Comment:
This comment doesn't really make any sense.

Reply:
Agree... Cut and paste...